components/brave_wallet/browser/brave_wallet_constants.h

```diff
@@ -42,6 +42,11 @@ constexpr char kSardineClientTokensURL[] =
     "https://api.sardine.ai/v1/auth/client-tokens";
 constexpr char kTransakURL[] = "https://global.transak.com/";
 constexpr char kTransakApiKey[] = "985d14f0-4cf5-4a4c-8917-78107620d3b7";
+constexpr char kSimpleHashUrl[] =
+    "https://api.simplehash.com";  // TODO (nvonpentz): replace with brave url
```

Comment:
Something of note, we should make sure this brave proxy URL is unique to wallet at least. We can reuse the proxy code from other places, but the domain should be unique enough to distinguish the network traffic is for the wallet rather than sharing the proxy with another feature like rewards. This is just a recent change we've decided to make after a H1 report about this being confusing.